src/google-maps/README.md

```diff
@@ -0,0 +1,127 @@
+
+## MapMarker
+
+The `MapMarker` component wraps the [`google.maps.Marker` class](https://developers.google.com/maps/documentation/javascript/reference/marker#Marker) from the Google Maps JavaScript API. The `MapMarker` component will display a marker on the map when it is a child of a `GoogleMap` component. Like the `GoogleMap` component, there is an `options` input as well as convenience inputs for `position`, `title`, `label`, and `clickable`, and supports all `google.maps.Marker` events as outputs.
+
+See the [example](#example) below or the [source](./map-marker/map-marker.ts) to read the API.
+
+## MapInfoWindow
+
+The `MapInfoWindow` component wraps the [`google.maps.InfoWindow` class](https://developers.google.com/maps/documentation/javascript/reference/info-window#InfoWindow) from the Google Maps JavaScript API. The `MapInfoWindow` has a `options` input as well as a convenience `position` input. Content for the `MapInfoWindow` is the inner HTML of the component, and will keep the structure and css styling of any content that is put there when it is displayed as an info window on the map.
+
+To display the `MapInfoWindow`, it must be a child of a `GoogleMap` component, and it must have its `open` method called, so a reference to `MapInfoWindow` will need to be loaded using the [`ViewChild` decorator](https://angular.io/api/core/ViewChild). The `open` method accepts an `MapMarker` as an optional input, if you want to anchor the `MapInfoWindow` to a `MapMarker`.
+
+See the [example](#example) below or the [source](./map-info-window/map-info-window.ts) to read the API.
+
+## Example
+
+```typescript
+// example-module.ts
+
+import {CommonModule} from '@angular/common';
+import {HttpClientJsonpModule, HttpClientModule} from '@angular/common/http';
+import {NgModule} from '@angular/core';
+import {GoogleMapsModule} from '@angular/google-maps';
+
+import {GoogleMapDemo} from './google-map-demo';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    GoogleMapsModule,
+    HttpClientJsonpModule,
+    HttpClientModule,
+  ],
+  declarations: [GoogleMapDemo],
+})
+export class GoogleMapDemoModule {
+}
+
+
+// example-app.ts
+import {HttpClient} from '@angular/common/http';
+import {Component, ViewChild} from '@angular/core';
+import {MapInfoWindow, MapMarker} from '@angular/google-maps';
+
+/** Demo Component for @angular/google-maps/map */
+@Component({
+  selector: 'google-map-demo',
+  templateUrl: 'google-map-demo.html',
+})
+export class GoogleMapDemo {
+  @ViewChild(MapInfoWindow, {static: false}) infoWindow: MapInfoWindow;
+
+  isReady = false;
+
+  center = {lat: 24, lng: 12};
+  markerOptions = {draggable: false};
+  markerPositions: google.maps.LatLngLiteral[] = [];
+  zoom = 4;
+  display?: google.maps.LatLngLiteral;
+
+  constructor(httpClient: HttpClient) {
+    httpClient.jsonp('https://maps.googleapis.com/maps/api/js?key=<YOUR_API_KEY>', 'callback')
```

Comment:
Why do this instead of adding a script tag to `index.html` for the basic example?

Reply:
My concern was that the basic example would not show how to make sure that the API was loaded before loading the Google Maps components. Do you think this would be okay if we included the link you suggested for loading the API? Another solution would be to use an index.html file as you suggested for the basic example, but also including an "advanced" example like this.

Reply:
Changed to use index.html as suggested.